src/EditorFeatures/Core/FindReferences/AbstractFindReferencesService.cs

```diff
@@ -54,7 +54,7 @@ internal abstract partial class AbstractFindReferencesService :
             return Tuple.Create(result, project.Solution);
         }
 
-        private static string GetDisplayName(ISymbol symbol)
+        public static string GetDisplayName(ISymbol symbol)
         {
             return symbol.IsConstructor() ? symbol.ContainingType.Name : symbol.Name;
```

Comment:
What about destructors, operators, etc.  Should this be some form of `Symbol.ToDisplayString()` instead?